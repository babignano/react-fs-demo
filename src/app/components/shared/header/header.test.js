import React from 'react';
import configureStore from 'redux-mock-store'
import Header from './header.js';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import { BrowserRouter as Router } from 'react-router-dom';


const mockStore = configureStore([])

describe('Header', () => {
    test('should match snapshot', () => {
        const component = renderer.create(
            <Provider store={mockStore()}>
                <Router>
                    <Header />
                </Router>
            </Provider>);
        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
    test('should display user email when in state', () => {

    });
});
