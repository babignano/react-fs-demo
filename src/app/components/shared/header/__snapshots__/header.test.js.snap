// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should match snapshot 1`] = `
<header>
  <nav>
    <a
      href="/user"
      onClick={[Function]}
    />
    <a
      href="/tasks"
      onClick={[Function]}
    >
      Tasks
    </a>
    <a
      href="/signout"
      onClick={[Function]}
    >
      Sign Out
    </a>
  </nav>
</header>
`;
